platform/ios/test/MGLAnnotationViewTests.swift

```diff
@@ -0,0 +1,23 @@
+import XCTest
+import Mapbox
+
+class CustomAnnotationView: MGLAnnotationView {
+    
+    override init(reuseIdentifier: String?) {
+        super.init(reuseIdentifier: reuseIdentifier)
+    }
+    
+    required init?(coder aDecoder: NSCoder) {
+        super.init(coder: aDecoder)
+    }
+    
+}
+
+class MGLAnnotationViewTests: XCTestCase {
```

Comment:
How is it that there are identically named MGLAnnotationViewTests class declarations in MGLAnnotationViewTests.m and MGLAnnotationViewTests.swift? Would it be possible to fashion the Swift class into an extension of the Objective-C class?